logic/transaction.js

```diff
@@ -144,8 +145,16 @@ Transaction.prototype.getBytes = function (trs, skipSignature, skipSecondSignatu
 			}
 		}
 
+
 		bb.writeLong(trs.amount);
 
+		if (trs.data) {
+			var dataBuffer = new Buffer(trs.data, 'utf8');
+			for (i = 0; i < 16; i++) {
+				bb.writeByte(dataBuffer[i] || 0);
+			}
+		} 
```

Comment:
I think we should increase the buffer size to at least 64. Typical use case would be for saving hashes. This is obviously debatable.